### PR TITLE
add Fcal

### PR DESCRIPTION
**Pull request by 5anthosh**
Adding new library to awesome list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -921,6 +922,7 @@ http://www.listjs.com
 * [Jaavascript tutorial](https://hackr.io/tutorials/learn-javascript) - Learn Javascript online from a diverse range of user ranked online tutorials.
 
 
+
```
Comment (gokaygurcan) [inline README.md]: why this space?

Comment (5anthosh) [reply README.md]: ho,  it might due to my editor markdown formater

Comment (gokaygurcan) [reply README.md]: nice, can you please remove it?

Comment (5anthosh) [reply README.md]: sure

Comment (gokaygurcan) [reply README.md]: thank you!

**Pull request merged**